lua/nordic/colors/init.lua

```diff
@@ -3,13 +3,17 @@ local P = require 'nordic.colors.nordic'
 
 local C = {}
 
-local function build_palette()
-    -- Make a new palette.
-    -- Copy the original palette this overrides C.
-    C = vim.deepcopy(P)
-    -- Because of the override we need to re-add the build_palette to the C table.
-    C.build_palette = build_palette
-    -- If we had attached build_palette to C we would lost it after the override.
+function C.build_palette()
+    -- clear all values except build_palette
+    for k, _ in pairs(C) do
+        if k ~= "build_palette" then
+            C[k] = nil
+        end
+    end
+    -- copy all values from the base palette
+    for k, v in pairs(vim.deepcopy(P)) do
+        C[k] = v
+    end
```

Comment:
Maybe we should move all of this out into a function?

Reply:
I did that when I merged it with `changing_palette`, but I didn't think of that here I will update it